src/platform/src/common/features/create-organization/components/CreateOrganizationForm.jsx

```
@@ -83,14 +83,11 @@ const CreateOrganizationForm = ({
-    setFormData((prev) => ({
-      ...prev,
-      organizationSlug: suggestion,
-    }));
+    setFormData((prev) => ({ ...prev, organizationSlug: suggestion }));
     setErrors((prev) => {
-      const c = { ...prev };
-      delete c.organizationSlug;
-      return c;
+      const copy = { ...prev };
+      delete copy.organizationSlug;
+      return copy;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)